traffic_portal/app/src/app.js

```diff
@@ -464,7 +464,7 @@ trafficPortal.factory('authInterceptor', function ($rootScope, $q, $window, $loc
                         messageModel.setMessages(alerts, true);
                         // forward the to the login page with ?redirect=page/they/were/trying/to/reach
                         $location.url('/login').search({ redirect: encodeURIComponent(url) });
-                    }, 200);
+                    }, 100);
```

Comment:
this whole timeout thing is weird. maybe it can be removed. i'll test it out. can't remember why i put it in here.

Reply:
When I remove it it also doesn't show up messages